patches/chrome-browser-ui-views-extensions-extension_popup.cc.patch

```diff
@@ -10,12 +10,12 @@ index db05a2b9b79e16562096916eea2f1a5362c09f34..7b35a6ea6abafa2bfeb538e292f85845
  #include "chrome/browser/chrome_notification_types.h"
  #include "chrome/browser/devtools/devtools_window.h"
  #include "chrome/browser/extensions/extension_view_host.h"
-@@ -67,8 +68,22 @@ gfx::Size ExtensionPopup::CalculatePreferredSize() const {
+@@ -66,7 +67,23 @@ gfx::Size ExtensionPopup::CalculatePreferredSize() const {
+ 
  void ExtensionPopup::AddedToWidget() {
    BubbleDialogDelegateView::AddedToWidget();
- 
--  const int radius =
--      GetBubbleFrameView()->bubble_border()->GetBorderCornerRadius();
+-  const int radius = GetBubbleFrameView()->corner_radius();
++
```

Comment:
careful about adding extra lines. Can you do a followup to convert this to a `DEFINE` so we can move the comments out of here too and also reduce to a single line?

Reply:
Addressed in c4d7da5